3.2ListOfContacts/ListOfContacts/ContactStorage.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace ListOfContacts
+{
+    public sealed class ContactStorage
+    {
+
+        private readonly List<string> keys = new List<string>()
```

Comment:
It is better just use constants like 
public const English = 'English' and so on,

Reply:
I think, I fixed this.